core/src/main/kotlin/core/model/JapaneseLyricsType.kt

```diff
@@ -10,7 +12,7 @@ enum class JapaneseLyricsType(
     KanaCv(false, true),
     KanaVcv(false, false);
 
-    fun findBestConversionTargetIn(outputFormat: Format): JapaneseLyricsType? {
+    fun findBestConversionTargetIn(@Suppress("NON_EXPORTABLE_TYPE") outputFormat: Format): JapaneseLyricsType? {
```

Comment:
Instead of suppressing the warning, could we make this function `internal` to hide it?

Reply:
No, it's used in app section so it will be compile error